packages/analytics/src/helpers.ts

```diff
@@ -185,6 +185,7 @@ async function gtagOnEvent(
     // if not all entries in the 'send_to' field could be mapped to
     // a FID. In these cases, wait on all pending initialization promises.
     if (initializationPromisesToWaitFor.length === 0) {
+      /* eslint-disable @typescript-eslint/no-floating-promises */
```

Comment:
This should have a corresponding eslint-enabled after the "errant" statement below.

Reply:
I replaced all instances of this with `eslint-disable-next-line` which has the same effect in a single comment.
I forgot `eslint-disable` was for entire files- thanks for reminding me!